### ISSUE TITLE
load-more-clickable is not allowed but documentation states otherwise

### ISSUE THREAD

**Issue by alabiaga** [ampproject/amphtml#22666]
https://amp.dev/documentation/components/amp-list#load-more-failed

The load-more service inserts the load-more-clickable along with service generated HTML if the load-more-failed attribute isn't present but in the documentation it seems to state that it is allowed to be manually set it up.

> ...To account for this, you can have a generally unclickable element with a button containing the load-more-clickable element...

Either the validator rules needs to be changed or the documentation is incorrect. Pls advice.

**Comment by alabiaga** [comment]
Ali, let me know if you have context on this and I can make the validator change if anything. We can also just wait till Cathy is back. Thanks

**Comment by aghassemi** [comment]
@alabiaga, yeah it is missing the in validation rules. We need a new rule that allows `load-more-clickable` on any child of `amp-list-load-more`. Up to your if you like to send a PR or wait for Cathy if not urgent.

**Comment by alabiaga** [comment]
@aghassemi  I can make the change. I am actually helping out a partner with this feature of the component so it'll be useful to understand it more. Many thanks.

**Comment by alabiaga** [comment]
cc\ @cathyxz @aghassemi
